.travis.yml

```diff
@@ -1,10 +1,18 @@
 language: python
-python:
-  - 2.7
+python: 2.7.13
+sudo: false
+cache:
+  pip: true
+  directories:
+    - eggs
+env:
+  - PLONE_VERSION=5.1.x
```

Comment:
What about also checking plone 5.0.x ? We are actually using this with plone 5.0.8.

Reply:
So you're suggesting to test both, aren't you?

Reply:
Ok, added both in the env